inc/utils/content.php

```diff
@@ -564,17 +581,19 @@ function get_categories_array($post){
 
  function odm_echo_extras($postID = "") {
  	 $postID = $postID ? $postID : get_the_ID();
-	 if (function_exists('get_post_meta')) :
-		 $get_author = get_post_meta($postID, 'author', true);
-		 $get_localized_author = get_post_meta($postID, 'author_'.odm_language_manager()->get_current_language(), true);
-	   if ($get_author != '' || $get_localized_author != ''):
-	     $news_source_info = '<span class="lsf">&#xE041;</span> ';
-	     if ($get_localized_author != ''):
-	         $news_source_info .= $get_localized_author.'<br />';
-	     else:
-	         $news_source_info .= $get_author.'<br />';
-	     endif;
-	   endif;
+	 if (function_exists('get_post_meta')):
```

Comment:
> What do you think of my suggestion to display the name of the source rather than the type of the source for announcements on the home page

@DBishton, the name of the source for announcement here, do you refer to "Embassy of Japan in Kingdom of Cambodia", right?  If you refer to this one, well, it makes sense for the announcement, but not the news. Let me clarify on this.
In news, it content sourced information including:
- Author : the author who write the article, not any one from OD team
- Sources: the names of newspaper eg. Phnom Penh  Post
- Link: the url link to original page of article

In Announcement:
- Author: the names of institution or organization eg. Embassy of Japan in the Kingdom of Cambodia
- Sources: the classification of publishers eg: Academic Institution, Government, Embassy, Civil Society / NGOs and so on
- Links: the urls to download the documentation
Hope this make sense to you. 

I would like to suggest to present them like this, what do you think?
![screenshot_2](https://cloud.githubusercontent.com/assets/6613530/19409101/456cd25e-92f7-11e6-9990-3426e7acb6da.png)